WalletWasabi.Fluent/Views/Wallets/WalletSettingsView.axaml

```diff
@@ -15,6 +15,25 @@
 
     <StackPanel Spacing="20" Margin="0 30">
 
+      <StackPanel Classes="settingsLayout">
+        <DockPanel>
+          <TextBlock Text="Name" />
+          <Button
+            Command="{Binding RenameCommand}"
+            Theme="{StaticResource AccentButton}"
+            DockPanel.Dock="Right">
+            <Button.Content>
+              <DockPanel>
+                <PathIcon Foreground="{DynamicResource AcrylicTrimForeground}" Margin="8 0 0 0" DockPanel.Dock="Right" Data="{StaticResource edit_regular}"/>
+                <TextBlock VerticalAlignment="Center" Text="{Binding WalletName, FallbackValue='Wallet name'}" />
```

Comment:
Can a long wallet name break the layout here?

Reply:
Fixed